tests/bundle.test.ts

```diff
@@ -0,0 +1,7 @@
+import { bundle } from "../mod.ts";
+import { assertEquals } from "./deps.ts";
+
+Deno.test("bundle (no error)", () => {
+  const result = bundle("mod.ts");
+  console.log(result);
```

Comment:
@divy-work the test should at least have some assertions using `assert` or `assertEquals`